File: IT/Variabili, operatori, strutture di controllo/Esercizi/C++/esercizio32.cpp
/* 
file esercizio32.cpp
compilare con: g++ esercizio32.cpp
*/

#include <iostream>

using std::cin;
using std::cout;
using std::endl;

int main()
{
	for (int i = 1; i <= 30; i++) {
		if (i % 5 == 0 & i % 3 == 0) {
			cout << "fizzbuzz ";
		}
		else if (i % 3 == 0) {
			cout << "fizz ";
		}
		else if (i % 5 == 0) {
			cout << "buzz ";
		}
		else {
			cout << i << " ";
		}
	}

	cout << endl;
}

File: IT/Variabili, operatori, strutture di controllo/Esercizi/Java/Esercizio32.java
/* file Esercizio32.java */

public class Esercizio32 {
	public static void main(String [] args) {
		for (int i = 1; i <= 30; i++) {
			if (i % 5 == 0 & i % 3 == 0) {
				System.out.print("fizzbuzz ");
			}
			else if (i % 3 == 0) {
				System.out.print("fizz ");
			}
			else if (i % 5 == 0) {
				System.out.print("buzz ");
			}
			else {
				System.out.print(i + " ");
			}
		}

		System.out.print(System.lineSeparator());
	}
}

